ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/tabsontop/v1/tabsontop/clientlibs/site/js/tabs.js

```diff
@@ -207,21 +190,35 @@
             }
         }
 
+        #bindEventsToTab(tabId){
+            var _self = this;
+            var tabs = this.#getCachedTabs();
+            var index = this.#getTabIndexById(tabId);
+            tabs[index].addEventListener("click", function(event) {
+                _self.#navigateAndFocusTab(tabId);
+            });
+            tabs[index].addEventListener("keydown", function(event) {
+                _self.#onKeyDown(event);
+            });
+        }
+
          /**
          * Returns the index of the active tab, if no tab is active returns 0
          *
          * @param {Array} tabs Tab elements
          * @returns {Number} Index of the active tab, 0 if none is active
          */
-        getActiveIndex(tabs) {
+        getActiveTab(tabs) {
```

Comment:
`getActiveTabIndex` 

getActiveTab looks like html el is returned.

Reply:
Yes renamed the method and the docs